CONTRIBUTING.md

```diff
@@ -88,6 +98,10 @@ You can find the necessary information about how to translate the game [here](do
 The planning board contains all issues and pull requests grouped
 by their priority and status. It can be found [here](https://github.com/orgs/Revolutionary-Games/projects/2).
 
+We have a separate planning board for only graphics tasks for artist
+to find things to work on:
+https://github.com/orgs/Revolutionary-Games/projects/4
```

Comment:
For consistency with the above?

```suggestion
to find things to work on. It can be found [here](https://github.com/orgs/Revolutionary-Games/projects/4).
```

Reply:
I kind of like the break form the monotony here by putting the full link...

Reply:
Sure, sounds fine to me.